rimage/transform/homography.go

```diff
@@ -11,7 +11,7 @@ import (
 type RawDepthColorHomography struct {
 	Homography   []float64 `json:"transform"`
 	DepthToColor bool      `json:"depth_to_color"`
-	RotateDepth  int       `json:"rotate_depth"`
+	RotateDepth  int       `json:"rotate_depth_degs"`
```

Comment:
sorry @bhaney , i missed one - i think `degs` is right based on how the field is used, but correct me if i'm wrong